### ISSUE TITLE
v1, v2, chros remain on loading spinner when no content scheduled

### ISSUE THREAD

**Issue by alanclayton** [Rise-Vision/viewer#137]
Viewer: 2-00-201609292007
Player: v1, v2, chrome os

Display: DPVU4JRSYPMX, CFG9X65JPEPP, more if needed.

Viewer just shows loading spinner indefinitely if the schedule it is assigned to is empty. Should show the standard msg that the display has no presentations or schedules scheduled.

**Comment by alex-deaconu** [comment]
@ezequielc @alanclayton investigated the issue, couple of things:
It is an existing issue not caused by the Viewer (verified that it occurs in the previous versions as well).

While I wasn't able to replicate by creating an empty schedule (either in the RVA or Apps), I could see the issue via an existing Schedule that was empty (https://apps-stage-0.risevision.com/schedules/details/ee454efb-56e3-4a22-bcd1-83edfeab39b3?cid=b428b4e8-c8b9-41d5-8a10-b4193c789443)

Basically, this particular Schedule has an **empty** `items:` field which makes the JSON object in the API response invalid resulting in a parsing error:
https://rvacore-test.appspot.com/v2/viewer/schedule/ee454efb-56e3-4a22-bcd1-83edfeab39b3

As opposed to a normal **empty** schedule where `items: []` which is valid JSON.

@alanclayton curious did you do something specific that I could follow to replicate this? Did you use RVA or Apps? 

Thanks!

**Comment by alanclayton** [comment]
@alex-deaconu I used Schedules App.

I had the Displays assigned to this Schedule https://goo.gl/lKnSPc. Once I finished my Viewer testing I removed the Displays form the Schedule.

**Comment by alex-deaconu** [comment]
@alanclayton while I wasn't able to identify how to replicate creating a Schedule with an empty field, I tested with existing Schedules and fixed the issue in the Viewer API.

This means both Displays & Previews will now show the warning as expected. An example of such a Schedule (note the `null` content field http://screencast.com/t/6ch5rbgB2):
http://preview.risevision.com/?type=schedule&id=3f4f2fd3-037a-4e56-9fd1-79387b4bfb4b&showui=false

Please verify. Thanks!

**Comment by alanclayton** [comment]
@alex-deaconu 👍
